### ISSUE TITLE
Add arguments to 'skip' groups of mobs in `/lm kill`

### ISSUE THREAD

**Issue by lokka30** [lokka30/LevelledMobs#156]
e.g. use [nametagged:false,tamed:true] in kill command to not kill nametagged mobs but kill tamed mobs. overrides config settings.

**Comment by lokka30** [comment]
See comments of #114

**Comment by lokka30** [comment]
Once ran, the command will print out what arguments were used, and the results of each argument. e.g.:
```
Killed 320 mobs:
- Worlds: world, world_nether, world_the_end
- Skipped nametagged mobs, affecting 135 mobs
- Skipped tamed mobs, affecting 3 mobs
```

**Comment by lokka30** [comment]
Implemented in LM 4.0
